docs/_includes/attr-doc.adoc

```diff
@@ -112,6 +126,65 @@ Another approach is to change the order of substitutions that are applied where
 The application is called {app-name}.
 ----
 
+==== Splitting attribute entry values over multiple lines
+
+===== Splitting long value strings
+
+Use backslashes `\` at the end of each attribute value to split long listings.
```

Comment:
I'll play around with the wording of this. 

I was thinking something like:

"Use a space and backslash ` \` at the end of each attribute...

.Why is a space required?
NOTE:  The space ensures lines are only split at the boundary of words, not mid-word.